packages/cxl-ui/scss/cxl-base-card.scss

```diff
@@ -8,7 +8,8 @@
   width: 100%;
   max-width: calc(100% - 2 * var(--lumo-space-l));
   height: max-content;
-  min-height: 270px;
+  min-height: 345px;
```

Comment:
I'm curious of the benefit of three definitions of height. Could you explain? :)

Reply:
well... there aren't any :) 
Now that we have managed to get fixed heights, only `height` makes sense. Also, it shouldn't be set in the base class, but overridden in the course card styles.